### PR TITLE
Set diagnosis key risk level based on day

### PR DESCRIPTION
**Pull request by tklingbeil**
* Risk level data from static vector

### CODE REVIEW

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/ProtoFormatConverterExtensions.kt [hunk 1]
```diff
@@ -9,18 +9,40 @@ object ProtoFormatConverterExtensions {
...
-            .setKeyData(ByteString.readFrom(it.keyData.inputStream()))
-            .setRollingStartIntervalNumber(it.rollingStartIntervalNumber)
-            .setRollingPeriod(ROLLING_PERIOD)
-            .setTransmissionRiskLevel(DEFAULT_TRANSMISSION_RISK_LEVEL)
-            .build()
+    fun List<TemporaryExposureKey>.transformKeyHistoryToExternalFormat() =
+        this.sortedWith(compareBy { it.rollingStartIntervalNumber })
+            .mapIndexed { index, it ->
+                val riskValue =
+                    if (index <= DEFAULT_TRANSMISSION_RISK_VECTOR.lastIndex) DEFAULT_TRANSMISSION_RISK_VECTOR[index]
```
Comment (ChristopherSchmitz) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/ProtoFormatConverterExtensions.kt:38]: @tklingbeil the key of today starts with TRANSMISSION_RISK_DAY_1  and not with TRANSMISSION_RISK_DAY_0. there is an offset to be apply.

**Review by ChristopherSchmitz — DISMISSED** [review]
please merge

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/ProtoFormatConverterExtensions.kt (hunk 1 above)
Comment (tklingbeil) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/ProtoFormatConverterExtensions.kt]: You're right. I have added an offset of 1.

**Comment by jakobmoellersap** [comment]
Pipeline Failing, see the  [circle report](https://948-268027139-gh.circle-artifacts.com/0/reports/ktlint/ktlintMainSourceSetCheck.txt)

**Review by jakobmoellerdev — APPROVED** [review]

**Pull request merged**